examples/webgl_mesh_batch.html

```diff
@@ -285,20 +294,18 @@
 		function sortFunction( list, camera ) {
 
 			// initialize options
-			this._options = this._options || {
+			const options = {
 				get: el => el.z,
-				aux: new Array( list.length ),
+				aux: aux,
```

Comment:
Yes I realized later that this list.length wasn't necessarily long enough on initialization for all cases 😅

One thing I wanted to do was keep this function self-contained so it's easy to copy-paste for user-world applications. Can we adjust this so the so the array is initialized to the full potential length here, instead?

Reply:
I did test this and it does re-allocate a lot during "warm-up" until the max-value reaches a big enough value, akin to not passing an auxiliary buffer at all. We also, don't have means of decreasing this buffer size, unlike the external solution which accomodates for when the geometry count changes.

While I understand your intention, I can't say I necessarily agree. But I will adjust my changes to what you prefer.

Reply:
I'm not sure I follow. The following approach will only allocate an array on the first run of the sort function per BatchedMesh instance:

```js
this._options = this._options || { aux: new Array( ... ) };
```

In order to allocate this on the first run for the worst case (sorting all elements) we can allocate an array with `this._maxGeometry`. Admittedly this should be exposed publicly on `BatchedMesh` so it's accessible - probably with a getter so it can't be adjusted.

```js
new Array( this._maxGeometry )
```

But perhaps I've misunderstood something?

Reply:
No, you are correct. This time it was me who misunderstood what you meant :)
Will patch the PR sortly, thanks for reviewing.